packages/main/src/plugin/color-registry.ts

```
@@ -458,53 +471,63 @@ export class ColorRegistry {
+      hcDark: purple[600],
     });
 
     this.registerColor(`${sNav}expander`, {
       dark: white,
       light: charcoal[700],
+      hcDark: white,
     });
   }
 
   protected initCardContent(): void {
     this.registerColor(`card-bg`, {
```

Biome (2.4.7)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)